README.md

```diff
@@ -16,7 +16,7 @@ The current focus is on nonlinear least squares with support for sparsity, batch
 
 ## Getting Started
 - Prerequisites
-    - We *strongly* recommend you install `theseus` in a venv or conda environment.
+    - We *strongly* recommend you install `theseus` in a venv or conda environment with Python <= 3.9.
```

Comment:
also >=3.7? So 3.7-3.9.
We can also update the tag at the top: [python 3.7, 3.8, 3.9]

Reply:
Added.